### ISSUE TITLE
Update new website immediately upon receipt of data with an alarm

### ISSUE THREAD

**Issue by ba58smith** [ba58smith/lora-receiver#11]
Short-circuit the web update delay if we get any new data packet that just went from "no alarm" to "alarm" status. That way, after an alarm situation is rec'd, it will show on the website almost immediately.

Do this only after switching to a different website than Jim's, so that I can properly highlight the alarm on the website.

**Comment by ba58smith** [comment]
No longer relevant, as all data is sent to InfluxDB / Grafana as soon as it's received.
